### PR TITLE
index.html for Pluto

### PR DESCRIPTION
**Pull request by rgetz**
Make the index.html page a little more dynamic, by adding
username/password, and the tupples for toolchains

Signed-off-by: Robin Getz <robin.getz@analog.com>

### CODE REVIEW

File: board/pluto/msd/img/version.js [hunk 1]
```diff
@@ -214,34 +216,74 @@ function libiio_url() {
...
-			document.getElementById('versionsection').className = "";
-			document.getElementById('plutsdr-fw-download').style.visibility = "hidden";
-			document.getElementById('hideupgrade').style.display = "none";
-		} else if (res == 0) {
-			message = "Pluto is using the same version as latest release!";
-			document.getElementById('versionsection').className = "";
-			document.getElementById('plutsdr-fw-download').style.visibility = "hidden";
-			document.getElementById('hideupgrade').style.display = "none";
+		var VerOnGithub = response[0].name;
+		var VerLocal = "0.30"; //"#BUILD#";
```
Comment (mhennerich) [inline board/pluto/msd/img/version.js]: Can you fix this?

File: board/pluto/msd/img/version.js [hunk 2]
```diff
@@ -214,34 +216,74 @@ function libiio_url() {
...
+			document.getElementById('sysroot').innerHTML = "Unknown SysRoot";
+			jQuery('#sysroot').removeAttr('href');
 		} else {
-			message = "Failure in comparing version, latest upstream is " + VerOnGithub;
-			document.getElementById('versionsection').className = "";
+			var res = versionCompare(VerLocal, VerOnGithub);
+			var message;
+			var sysroot_message;
+			var j;
+			vat k;
```
Comment (mhennerich) [inline board/pluto/msd/img/version.js]: I guess this should be
var k;

**Comment by mhennerich** [comment]
I fixed the issues - rebased and froce pushed again.

**Comment by rgetz** [comment]
Thanks for the fix.

I had run things through JSLint,  and obviously made a mistake. 

I will re-based based on the latest merge.

**Comment by rgetz** [comment]
replaced by #27

**Pull request closed without merging**